src/service/wallet.ts

```diff
@@ -1,4 +1,4 @@
-/** This service is a .tsx so it can present contentful toasts. */
```

Comment:
consider squashing this commit into the one that made this line so it doesn't exist in master history.

Reply:
👍Done